### ISSUE TITLE
Implement "Backchannel" to upload bolus/carb/etc. data to clarity (original dexcom api does not allow that).

### ISSUE THREAD

**Issue by juehv** [DiaKEM/dexcom-g7-apk-patcher#51]
Would be great to have my data at clarity, as many doctors prefert this charts instead of nightscout reporter.

**Comment by khskekec** [comment]
Is this supported by clarity? Could you provide a screenshot to give me more context?

**Comment by juehv** [comment]
Yes, within the dexcom app you can report bolus, carbs etc. See below:
![Screenshot_20231107-144227](https://github.com/DiaKEM/dexcom-g7-apk-patcher/assets/637973/8ff1b21d-e343-4cc4-bc04-e5d249694e9a)
![Screenshot_20231107-144255](https://github.com/DiaKEM/dexcom-g7-apk-patcher/assets/637973/84eece60-b07f-4cf3-a75b-b01edfb240d3)


But I realised no data is acually uploaded to dexcom clarity. Is that already patched out? Or is it a issue with my setup?
